app/javascript/elm/tests/MainTests.elm

```diff
@@ -501,6 +501,56 @@ toggleIndoorFilter =
         ]
 
 
+toggleStreamingFilter : Test
+toggleStreamingFilter =
+    describe "active/dormant filter tests:"
+        [ test "toggle is displayed" <|
+            \_ ->
+                { defaultModel | page = Fixed }
+                    |> view
+                    |> Query.fromHtml
+                    |> Expect.all
+                        [ Query.has [ Slc.attribute <| ariaLabel "active" ]
+                        , Query.has [ Slc.attribute <| ariaLabel "dormant" ]
+                        ]
+        , test "active is selected by default" <|
+            \_ ->
+                { defaultModel | page = Fixed }
+                    |> view
+                    |> Query.fromHtml
+                    |> Query.find [ Slc.attribute <| ariaLabel "active" ]
+                    |> Query.has [ Slc.attribute <| class "toggle-button--pressed" ]
+        , test "clicking dormant button triggers ToggleStreaming" <|
+            \_ ->
+                { defaultModel | page = Fixed }
+                    |> view
+                    |> Query.fromHtml
+                    |> Query.find [ Slc.attribute <| ariaLabel "dormant" ]
+                    |> Event.simulate Event.click
+                    |> Event.expect ToggleStreaming
+        , test "clicking active button triggers ToggleStreaming" <|
+            \_ ->
+                { defaultModel | page = Fixed }
+                    |> view
+                    |> Query.fromHtml
+                    |> Query.find [ Slc.attribute <| ariaLabel "active" ]
+                    |> Event.simulate Event.click
+                    |> Event.expect ToggleStreaming
+        , test "when isStreaming is true ToggleStreaming triggers Ports.toggleStreaming with False" <|
+            \_ ->
+                { defaultModel | page = Fixed }
+                    |> update ToggleStreaming
+                    |> Tuple.second
+                    |> Expect.equal (Ports.toggleStreaming False)
+        , test "when isStreaming is false ToggleStreaming triggers Ports.toggleStreaming with False" <|
```

Comment:
I guess this and the previous test are using the same case (`isStreaming = True`) cause that's the default in `defaultModel`

Reply:
yes, you're totally right, mixup